tests/resources/algorithms/gpr.py

```diff
@@ -0,0 +1,103 @@
+#
+# Copyright 2022 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import json
```

Comment:
I would much rather have the algos in different repos. Or a "resources/examples" repo. That way we don't inject unneeded dependencies into ocean.py, and it is clearer as a user flow that these are independent. They are independently-published algorithms, not part of ocean.py.

Reply:
Okay. I'll add them to https://github.com/oceanprotocol/cdt-algorithms

Reply:
It appears I don't have permission to submit PRs to https://github.com/oceanprotocol/cdt-algorithms, I think, because it's not part of the Engineering group. https://github.com/orgs/oceanprotocol/teams/engineering.  I've sent a message to the team via Slack to try and get it added.